ods_ci/tests/Tests/1300__model_registry/1302_model_registry_model_serving.robot

```diff
@@ -0,0 +1,277 @@
+*** Settings ***
+Documentation     Test suite for Model Registry Integration
+Suite Setup       Prepare Model Registry Test Setup
+Suite Teardown    Teardown Model Registry Test Setup
+Library           OperatingSystem
+Library           Process
+Library           OpenShiftLibrary
+Resource          ../../Resources/Page/ODH/JupyterHub/HighAvailability.robot
+Resource          ../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource          ../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+
+
+*** Variables ***
+${PRJ_TITLE}=                        model-registry-project-tony
+${PRJ_DESCRIPTION}=                  model resgistry test project
+${AWS_BUCKET}=                       ${S3.BUCKET_2.NAME}
+${WORKBENCH_TITLE}=                  registry-wb
+${DC_S3_NAME}=                       model-registry-connection
+${MODELREGISTRY_BASE_FOLDER}=        tests/Resources/CLI/ModelRegistry
+${EXAMPLE_ISTIO_ENV}=                ${MODELREGISTRY_BASE_FOLDER}/samples/istio/components/example_istio.env
+${ISTIO_ENV}=                        ${MODELREGISTRY_BASE_FOLDER}/samples/istio/components/istio.env
+${SAMPLE_ONNX_MODEL}=                ${MODELREGISTRY_BASE_FOLDER}/mnist.onnx
+${SERVICE_MESH_MEMBER}=              ${MODELREGISTRY_BASE_FOLDER}/serviceMeshMember.yaml
+${ENABLE_REST_API}=                  ${MODELREGISTRY_BASE_FOLDER}/enable_rest_api_route.yaml
+${IPYNB_UPDATE_SCRIPT}=              ${MODELREGISTRY_BASE_FOLDER}/updateIPYNB.py
+${CERTS_DIRECTORY}=                  certs
+${MODEL_REGISTRY_DB_SAMPLES}=        ${MODELREGISTRY_BASE_FOLDER}/samples/secure-db/mysql-tls
+${JUPYTER_NOTEBOOK}=                 MRMS_UPDATED.ipynb
+${JUPYTER_NOTEBOOK_FILEPATH}=        ${MODELREGISTRY_BASE_FOLDER}/${JUPYTER_NOTEBOOK}
+${DC_S3_TYPE}=                       Object storage
+${NAMESPACE_ISTIO}=                  istio-system
+${NAMESPACE_MODEL-REGISTRY}=         odh-model-registries
+${SECRET_PART_NAME_1}=               modelregistry-sample-rest
+${SECRET_PART_NAME_2}=               modelregistry-sample-grpc
+${SECRET_PART_NAME_3}=               model-registry-db
+
+
+*** Test Cases ***
+Verify Model Registry Integration With Secured-DB
+    [Documentation]    Verifies the Integartion of Model Registry operator with Jupyter Notebook
+    [Tags]    OpenDataHub
+    Create Workbench    workbench_title=${WORKBENCH_TITLE}    workbench_description=Registry test
+    ...                 prj_title=${PRJ_TITLE}    image_name=Minimal Python  deployment_size=Small
+    ...                 storage=Persistent   pv_existent=${NONE}
+    ...                 pv_name=${NONE}  pv_description=${NONE}  pv_size=${NONE}
+    Workbench Should Be Listed      workbench_title=registry-wb
+    Open Data Science Project Details Page       project_title=${PRJ_TITLE}
```

Comment:
Aren't you already in this page?

Reply:
I'm not sure what you mean?

Reply:
You call the keyword `Open Data Science Project Details Page       project_title=${PRJ_TITLE}`, but given the code executed between the Suite Setup and this Test Case, aren't you already in the Data Science Project details page at this point?